### PR TITLE
[Created] data: adityassharma-ss.json

### PR DESCRIPTION
**Pull request by adityassharma-ss**
Added My details, links & milestones in the data folder.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2543"></a>

### CODE REVIEW

File: data/adityassharma-ss.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Aditya Sharma",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full Stack Developer | DevOps | Open Source",
+    "avatar": "https://github.com/adityassharma-ss.png",
+    "tags": ["Java", "JavaScript", "ReactJS", "DevOps", "Open Source"],
+    "socials": [
+      { "icon": "twitter", "url": "https://twitter.com/aadityatwt" },
+      { "icon": "github", "url": "https://github.com/adityassharma-ss" }
```
Comment (eddiejaoude) [inline data/adityassharma-ss.json:10]: Looks good, just one small change, these icons need to be in the same format used in links section - do the docs need updating as a few people have done something similar?

Comment (adityassharma-ss) [reply data/adityassharma-ss.json:10]: Yes. I think so because i just copy pasted the docs here. Nevermind, I'm gonna fix that.

**Review by senali-d — COMMENTED** [review]
Now every thing look good.

File: data/adityassharma-ss.json (hunk 1 above)
Comment (eddiejaoude) [reply data/adityassharma-ss.json]: ok thanks, can you let me know which docs you used? I will try to search for it so we can fix the docs

Comment (eddiejaoude) [reply data/adityassharma-ss.json]: I found it, I have fixed the docs now. Thank you, sorry for the confusion

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**